scripts/giphy.coffee

```diff
@@ -0,0 +1,43 @@
+# Description:
+#   A way to search images on giphy.com.  Forked from 
+#
+# Configuration:
+#   HUBOT_GIPHY_API_KEY
+#
+# Commands:
+#   hubot animate me <query> - Returns an animated gif matching the requested search term.
+
+giphy =
+  api_key: process.env.HUBOT_GIPHY_API_KEY
+  base_url: 'http://api.giphy.com/v1'
+  no_gif_found_query: process.env.HUBOT_GIPHY_DEFAULT_QUERY or 'otter'
+
+module.exports = (robot) ->
+  robot.respond /(animate|gif|giphy)( me)? (.*)/i, (msg) ->
+    giphyMe msg, msg.match[3], (url) ->
+      msg.send url
+
+giphyMe = (msg, query, cb) ->
+  endpoint = '/gifs/search'
```

Comment:
Do we want to use the [Translate endpoint](https://github.com/giphy/GiphyAPI#translate-endpoint) instead? It looks like it's designed for integrations like this.